### PR TITLE
Added e-book in several places and other minor changes

### PR DESCRIPTION
**Pull request by GeorgeKerscher**
It seems that the EAA uses e-book and e-readers in the language of the directive. I introduced these terms and use e-book and digital publication interchangeably. I have not defined these terms or made a statement that they are used interchangeably.

If this is needed, let me know.

I also looked for areas where we could make it less EPUB centric. I made some small changes. It seems thatPDF, or EPUB are neutral, until you come to the conformance section. We do not have an example of a PDF. I would need some help in writing such an example.

Perhaps a PDF conformance statement should be added after we have a techniques section to support internal PDF metadata. Alternatively,it might be possible to craft a conformance statement for PDF from the ONIX metadata.

### CODE REVIEW

**Comment by mattgarrish** [comment]
W3C style is to US spellings and ebook (or eBook) is more common, at least that I've ever found in publishing retail. Hyphenated "e" things make me want to party like it's 1999.

**Comment by GeorgeKerscher** [comment]
Based on this comment and knowing that it is a broad not only W3c audience, I am inclinedd to add a note. It would clarify that 
digital publications are descriedd in many ways, including: e-books, eBooks, EPUB, PDF publications, audio books, DAISY books,  PDF publications, and more. It should clarify that it is a digital version that is intended to be used through a computer or a device, and that persons with disabilities use Assistive Technologies to access this information through mainstream Reading Systems or through technologies optimized for their .particular disability.

**Comment by mattgarrish** [comment]
The benefit of digital publication is that it avoids the book baggage, specifically being tied to text. Since it's in the title of the document, we should be consistent in using that term unless there's a specific case where we need to call out a particular format. A definition that clarifies it's an overarching terms for all the types of publications you've listed makes sense to me.

**Review by gregoriopellegrino — APPROVED** [review]

**Review by clapierre — APPROVED** [review]
Like Matt says there are many places here where e-book is present also e-reading device.
Should this also be changed to eReading device? or just electronic reading device?

**Comment by mattgarrish** [comment]
Isn't eReader the more common US name?

**Comment by clapierre** [comment]
Hi Matt, I believe so. So George, I think it would be best to change e-Reading device to eReader. which in theory you could say

eReader (electronic reading device) if you wanted to.

File: UX-Guide-Metadata/draft/principles/index.html [hunk 1]
```diff
@@ -130,22 +130,22 @@ <h2>Introduction</h2>
...
-			<p>The good news is more and more publishers are creating e-books that are Born Accessible (i.e.,
+			<p>The good news is more and more publishers are creating digital publications that are Born Accessible (i.e.,
 				accessible from the outset, not fixed later) and getting the accessibility validation or audit done by
 				independent organizations.</p>
 		</section>
 		<section id="general-overview">
 			<h2>General overview</h2>
 
-			<p>This document helps those who wish to render accessibility metadata directly to users understand how to
+			<p>These guidelines   helps those who wish to render accessibility metadata directly to users understand how to
```
Comment (mattgarrish) [inline UX-Guide-Metadata/draft/principles/index.html:140]:
```suggestion
			<p>These guidelines   help those who wish to render accessibility metadata directly to users understand how to
```

**Pull request merged**